capybaraKV/README.md

```diff
@@ -18,30 +18,37 @@ repo in the `osdi25-artifact` branch.
 
 1. Install Verus from `https://github.com/verus-lang/verus`.
 2. If you want to do line counting, run `cargo install tokei` and `pip install prettytable`.
+3. **Optional**: If you are on Linux (or WSL2) and would like to run CapybaraKV on (real or emulated) PM via Linux's DAX feature, run the following to install PMDK to access PM and LLVM/Clang to generate Rust bindings to PMDK: `sudo apt install libpmem1 libpmemlog1 libpmem-dev libpmemlog-dev llvm-dev clang libclang-dev`
 
 ### Verification
 
-To verify CapybaraKV, run the following. If you don't plan to do line counting as discussed in the [next section](#computing-code-size), you can leave off the `--emit=dep-info` from the last command.
+To verify CapybaraKV, make sure Verus (usually at `verus/source/target-verus/release`) is in your path and run one of the following, depending on your setup:
```

Comment:
I suggest clarifying that each of these command lines is meant to be run from the `capybarakv` subdirectory of this `capybaraKV` directory.